Makefile

```diff
@@ -1,5 +1,6 @@
 export GOPATH ?= $(shell go env GOPATH)
 export GO111MODULE ?= on
+export GOPRIVATE ?= github.com/insolar
```

Comment:
why is it need here?

Reply:
added private repo https://github.com/insolar/block-explorer/pull/21/files#diff-37aff102a57d3d7b797f152915a6dc16R11
This line enables you to download repositories locally with `go mod download`. While we work with vendor it is necessary. On CI downloading modules will not succeed, but there is no downloading modules so it's only for local usage.